src/main/resources/templates/index.html

```diff
@@ -7,12 +7,56 @@
   <th:block th:replace="fragments/demoBanner :: demoBanner"/>
   <main id="content" role="main" class="slab slab--white spacing-below-0">
     <section class="slab">
-      <div class="grid">
-        <div class="width-one-half">
-          <h1 class="spacing-below-35 spacing-above-5" th:text="#{index.title}"></h1>
-          <a id="upload_docs" href="/laDocUpload/clientInfo" th:text="#{index.upload-documents}" class="button"></a>
-        </div>
+      <h1 class="spacing-below-35 spacing-above-5" th:utext="#{index.header}"></h1>
+      <p th:utext="#{index.upload-description}"></p>
+      <div class="display-flex flex-column">
+        <a id="upload_docs_1" th:href="#{index.upload-documents-primary.link}"
+           th:text="#{index.upload-documents-primary}" class="button button--primary"></a>
+        <a id="upload_docs_2" th:href="#{index.upload-documents-second.link}"
+           th:text="#{index.upload-documents-second}" class="button"></a>
+        <a id="upload_docs_3" th:href="#{index.upload-documents-third.link}"
+           th:text="#{index.upload-documents-third}" class="button"></a>
       </div>
+
+      <p th:text="#{index.these-documents}"></p>
+          <table>
+            <tr>
+              <td th:utext="#{index.snap-info}"></td>
+              <td th:utext="#{index.fitap-info}"></td>
+            </tr>
+            <tr>
+              <td th:utext="#{index.kcsp-info}"></td>
+              <td th:utext="#{index.lcap-info}"></td>
+            </tr>
+          </table>
+      <p th:utext="#{index.looking-for-other-programs}"></p>
+      <a href="/laDocUpload/clientInfo/" th:text="#{index.get-started}" class="button button--primary"></a>
+
+      <h2 th:text="#{index.heres-a-quick-list}"></h2>
+      <p th:utext="#{index.you-can-upload}"></p>
+
+      <ul class="list--checkmark">
```

Comment:
The checkmark emojis dont seem to be loading

Reply:
For now, I have added it to our project but it is probably worth looking into why the background image was not found from the honeycrisp provided by form-flow library